schedulearn-frontend/src/components/WorkerList.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import './WorkerList.css';
+
+export interface Item {
+  name: string;
```

Comment:
These fields should be the following, due to dotnetify:
UserId
FullName
JobTitle
DateFrom
DateTo